src/translations/screens/Tickets.ts

```diff
@@ -7,6 +7,9 @@ const TicketsTexts = {
   header: {
     title: _('Billetter', 'Tickets'),
   },
+  dashboard: {
+    label: _('Kjøp billetter', 'Buy tickets'),
+  },
```

Comment:
I think this is the wrong translation file to put this text, as it is used in Dashboard components and not in Tickets components.

Reply:
Fixed :)